### ISSUE TITLE
R16 and Chrome-50 SSL problems

### ISSUE THREAD

**Issue by mmzeeman** [zotonic/zotonic#1264]
There is a problem with R16 and Chrome-50. This causes ssl connectivity problems with Chrome 50.

The only way to fix this is either upgrading to erlang 17 or 18, or apply a hotfix to R16. The hotfix makes it possible for clients to use newer elliptic curve algorithms.

The hotfix is:

`lib/ssl/src/tls_v1.erl` around rule 428 in `enum_to_oid/1`. Add the line `enum_to_oid(_) -> undefined.`

```
...
425 enum_to_oid(22) -> ?secp256k1;
426 enum_to_oid(23) -> ?secp256r1;
427 enum_to_oid(24) -> ?secp384r1;
428 enum_to_oid(25) -> ?secp521r1;
429 enum_to_oid(_) -> undefined.
```

Note: R16 will not be maintained when 19 is released in June.

**Comment by ddeboer** [comment]
Quick poll: who is still dependent on R16? We’ve decided to support R18 upwards on master, but perhaps we can drop R16 earlier on 0.x?

**Comment by ArthurClemens** [comment]
not me

**Comment by mworrell** [comment]
I am using R16 in production, but not with the newest release.
Can start upgrading if we fix #1263

**Comment by mmzeeman** [comment]
I'm using it right now. Currently busy testing 18.3. This looks ok after a little fix  in mochiweb. After that no more R16 for us. 

17 has ssl issues which are fixed in 18.1

Verstuurd vanaf mijn iPhone

**Comment by mworrell** [comment]
We don't support R16 anymore and this issue does not occur on OTP18+
